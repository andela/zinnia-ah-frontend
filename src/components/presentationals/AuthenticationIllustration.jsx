import React from 'react';

import Title from './Title.jsx';
import Illustration from '../../assets/images/reading-lady.svg';
import './AuthenticationIllustration.scss';

const AuthenticationIllustration = () => {
  const body = {
    fontSize: '20px',
    lineHeight: '24px',
    marginTop: '18px',
  };
  return (
    <div className="left">
      <Title title={title} />
      <p style={body}>
        A community of like minded authors to foster inspiration and innovation
        by leveraging the modern web.
      </p>
      <div className="illustration-container">
        <img
          src={Illustration}
          alt="A lady sitting on a pile of books and reading a book"
        />
      </div>
    </div>
  );
};

const title = {
  content: 'A Safe Haven For The Creative At Heart',
  class: 'title-lg',
};

export default AuthenticationIllustration;
