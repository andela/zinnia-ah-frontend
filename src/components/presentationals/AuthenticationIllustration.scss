.illustration-container {
  width: 100%;
  display: flex;
  justify-content: center;
  img {
    width:65%;
    height: 100%;
    margin-top: 18px;
  }
}
.left{
  margin-top: 54px;
  padding-right: 100px;
}
