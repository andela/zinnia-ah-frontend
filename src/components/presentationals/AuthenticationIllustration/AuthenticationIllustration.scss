.illustration-container {
  display: flex;
  justify-content: center;
  width: 100%;

  img {
    height: 100%;
    margin-top: 18px;
    width: 65%;
  }
}

.left {
  margin-top: 54px;
  padding-right: 100px;
}
