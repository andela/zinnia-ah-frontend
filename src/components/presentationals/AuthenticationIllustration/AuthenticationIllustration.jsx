import React from 'react';

import Title from '../Title/Title';
import Illustration from '../../../assets/images/reading-lady.svg';
import './AuthenticationIllustration.scss';

const AuthenticationIllustration = () => {
  const styles = {
    fontSize: '20px',
    lineHeight: '24px',
    marginTop: '18px',
  };
  return (
    <div className="left">
      <Title
        className="title-lg"
        content="A Safe Haven For The Creative At Heart"
      />
      <p style={styles}>
        A community of like minded authors to foster inspiration and innovation
        by leveraging the modern web.
      </p>
      <div className="illustration-container">
        <img
          src={Illustration}
          alt="A lady sitting on a pile of books and reading a book"
        />
      </div>
    </div>
  );
};

export default AuthenticationIllustration;
