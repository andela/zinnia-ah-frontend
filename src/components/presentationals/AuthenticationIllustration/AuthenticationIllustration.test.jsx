import React from 'react';
import { mount } from 'enzyme';

import AuthenticationIllustration from './AuthenticationIllustration';

describe('AuthenticationIllustration', () => {
  it('should render correctly', () => {
    const component = mount(<AuthenticationIllustration />);
    expect(component).toMatchSnapshot();
  });
});
