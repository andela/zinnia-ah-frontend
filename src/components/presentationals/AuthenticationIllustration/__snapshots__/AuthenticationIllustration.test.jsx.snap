// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthenticationIllustration should render correctly 1`] = `
<AuthenticationIllustration>
  <div
    className="left"
  >
    <Title
      className="title-lg"
      content="A Safe Haven For The Creative At Heart"
    >
      <h1
        className="title title-lg"
      >
        A Safe Haven For The Creative At Heart
      </h1>
    </Title>
    <p
      style={
        Object {
          "fontSize": "20px",
          "lineHeight": "24px",
          "marginTop": "18px",
        }
      }
    >
      A community of like minded authors to foster inspiration and innovation by leveraging the modern web.
    </p>
    <div
      className="illustration-container"
    >
      <img
        alt="A lady sitting on a pile of books and reading a book"
        src=""
      />
    </div>
  </div>
</AuthenticationIllustration>
`;
